hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -0,0 +1,539 @@
+"""
+Script to generate a dataset and perform hand-eye calibration using a Universal Robot UR5e robot.
+The script communicates with the robot through TCP/IP. The host is defined by the user.
```

Comment:
The host is defined by the user. It is not clear what this means. Remove it if it is not important or further explain it, e.g. put robot IP address in brackets.

Reply:
done